cli/src/funTest/kotlin/ExamplesTest.kt

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit
+
+import io.kotest.assertions.throwables.shouldNotThrow
+import io.kotest.assertions.withClue
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+import java.io.File
+import java.io.IOException
+
+import org.ossreviewtoolkit.evaluator.Evaluator
+import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.PackageCuration
+import org.ossreviewtoolkit.model.config.CopyrightGarbage
+import org.ossreviewtoolkit.model.config.RepositoryConfiguration
+import org.ossreviewtoolkit.model.config.Resolutions
+import org.ossreviewtoolkit.model.licenses.LicenseConfiguration
+import org.ossreviewtoolkit.model.readValue
+import org.ossreviewtoolkit.model.utils.SimplePackageConfigurationProvider
+import org.ossreviewtoolkit.reporter.reporters.AbstractNoticeReporter
+
+class ExamplesTest : StringSpec() {
+    private val examplesDir = File("../docs/examples")
+
+    init {
+        "ort.yml examples are parsable" {
+            examplesDir.listFiles { file ->
+                file.name.endsWith(".ort.yml")
+            }!!.forEach { file ->
```

Comment:
How about using `.orEmpty()` instead of `!!`?

Reply:
I'd rather have the test fail if it cannot find the files.

Reply:
Ok, just note that `listFile` returns `null` "if this abstract pathname does not denote a directory, or if an I/O error occurs", but not if there are no matching files.